packages/unmock-fetch/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": "0.3.8",
+  "name": "unmock-fetch",
+  "displayName": "unmock-fetch",
+  "main": "dist/index.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unmock/unmock-js.git"
+  },
+  "homepage": "https://github.com/unmock/unmock-js",
+  "bugs": {
+    "url": "https://github.com/unmock/unmock-js/issues"
+  },
+  "dependencies": {
+    "@types/debug": "^4.1.5",
+    "@types/url-parse": "^1.4.3",
```

Comment:
Should `debug` and its `@types` be part of the dependencies? (and not dev dependencies?)

Reply:
Yes, a library should include the types it uses.

Reply:
Of course, but `debug` is not a dependency. The code doesn't *need* it to function properly, so shouldn't it be part of dev dependencies?

Reply:
But `debug` is a dependency of `unmock-fetch` :) It's true that user's type-checking probably would pass fine even if `@types/debug` wasn't part of the dependencies, but it's generally just bad practice to leave out the types. We've had two breakages in releases already because of leaving out types for `sinon` and `node-fetch`, requiring users to manually add the types.